doc/releases/changelog-dev.md

```diff
@@ -4,34 +4,125 @@
 
 <h3>New features since last release</h3>
 
-* Devices have a new capability flag `capabilities()["supports_broadcasting"]`
-  and are now able to handle broadcasting of tapes.  In addition, the tape transform
-  `broadcast_expand` was added, which allows a tape that uses broadcasting 
-  (`tape.batch_size!=None`) to be expanded into multiple tapes without a broadcast dimension.
+* Parameter broadcasting within operations and tapes was introduced.
+
```

Comment:
```suggestion
* Parameter broadcasting within operations and tapes was introduced.
```

Reply:
:thinking: Not sure what happened here? :)

Reply:
oh weird, I was trying to delete the line break between the sentence and the PR links :)